packages/mdc-textfield/_mixins.scss

```diff
@@ -63,36 +63,79 @@
   }
 }
 
+@mixin mdc-text-field-bottom-line-color($color) {
+  &:not(.mdc-text-field--disabled) {
+    @include mdc-text-field-bottom-line-color_($color);
+  }
+}
+
+@mixin mdc-text-field-bottom-line-hover-color($color) {
+  &:not(.mdc-text-field--disabled) {
+    @include mdc-text-field-bottom-line-hover-color_($color);
+  }
+}
+
+@mixin mdc-text-field-bottom-line-focused-color($color) {
+  &:not(.mdc-text-field--disabled) {
+    @include mdc-text-field-bottom-line-focused-color_($color);
+  }
+}
+
+@mixin mdc-text-field-label-ink-color($color) {
```

Comment:
you can just say `label-color` there is only one way to color the label

Reply:
`.mdc-text-field--textarea .mdc-text-field__label` has a background-color attribute.